source/extensions/quic_listeners/quiche/platform/quic_logging_impl.h

```diff
@@ -82,6 +84,13 @@
 #define QUIC_NOTREACHED_IMPL() NOT_REACHED_GCOVR_EXCL_LINE
 #endif
 
+#define DCHECK_GT(a, b) DCHECK((a) > (b))
+#define DCHECK_GE(a, b) DCHECK((a) >= (b))
+#define DCHECK_LT(a, b) DCHECK((a) < (b))
+#define DCHECK_LE(a, b) DCHECK((a) <= (b))
+#define DCHECK_NE(a, b) DCHECK((a) != (b))
+#define DCHECK_EQ(a, b) DCHECK((a) == (b))
```

Comment:
I see no calls to these in the PR either. Just remove them?

Reply:
They are used in the quiche code introduced by the newly-added 2 quiche build targets: quiche_quic_core_base, quiche_quic_core

Reply:
Cool -- can you comment that that's what those are for?

Otherwise, at least in the context of this PR, they look unused.

Reply:
Added a comment above `#define CHECK`